docs/Java-SpringBoot-Developer.md

```diff
@@ -0,0 +1,369 @@
+# Java/Spring Boot Developer Documentation
+
+## Index:
+1. [Managed Identity and Key Vault](#managed-identity-and-key-vault)
+    - [Key Vault](#key-vault)
+2. [Key Rotation](#key-rotation) - WIP
+3. [Cosmos DB](#cosmos-db)
+    - [Spring Repository Pattern](#spring-repository-pattern)
+    - [Partition Key Function](#partition-key-usage-in-spring)
+4. [AKS Pod Identity Support](#aks-pod-identity-support) - WIP
+5. [Versioning](#versioning)
+6. [Application Insights](#application-insights)
+
+
+## Managed Identity and Key Vault
+
+After creating a Managed Identity for the Helium web app and assigning get and list secret permissions to Key Vault, the following code successfully authenticates using Managed Identity to create the Key Vault Client. Leveraging Managed Identity in this way eliminates the need to store any credential information in app code. 
+
```

Comment:
We should update the description for managed identity (or somewhere) to better reflect the current solution/state in spring boot.  The description as is says there's no need to store any credential information, but below it's called out that client id and key need to be provided to run locally. 

Maybe (as you suggested earlier), it's best to just remove that first paragraph for now?

Reply:
As I had mentioned, this documented is designed in a way similar to CSharp minus the language specifics. So first paragraph is describe the feature implementation from specs, second paragraph is about the gaps in SDK uncovered , feedbacks given and this paragraph is about alternative solutions used due to the gaps. IMO this should stay as that is feature description from specs and we are called out the gaps in the next section. Happy to get feedbacks from others as well.

Reply:
That works, I also notice now this paragraph only talks about the web app solution.